tests/neg/i11225.check

```diff
@@ -0,0 +1,49 @@
+-- Error: tests/neg/i11225.scala:5:16 ----------------------------------------------------------------------------------
+5 |  val x1: Int = uninitialized // error
+  |                ^^^^^^^^^^^^^
+  |                `uninitialized` can only be used as the right hand side of a mutable field definition
+-- Error: tests/neg/i11225.scala:6:28 ----------------------------------------------------------------------------------
+6 |  var x2: Int = if ??? then uninitialized else uninitialized // error // error
+  |                            ^^^^^^^^^^^^^
+  |                            `uninitialized` can only be used as the right hand side of a mutable field definition
+-- Error: tests/neg/i11225.scala:6:47 ----------------------------------------------------------------------------------
+6 |  var x2: Int = if ??? then uninitialized else uninitialized // error // error
+  |                                               ^^^^^^^^^^^^^
+  |                             `uninitialized` can only be used as the right hand side of a mutable field definition
+-- Error: tests/neg/i11225.scala:7:29 ----------------------------------------------------------------------------------
+7 |  var x3: Int = if true then uninitialized else 1 // error
+  |                             ^^^^^^^^^^^^^
+  |                             `uninitialized` can only be used as the right hand side of a mutable field definition
+-- Error: tests/neg/i11225.scala:9:28 ----------------------------------------------------------------------------------
+9 |  var x5: () => Int = () => uninitialized // error
+  |                            ^^^^^^^^^^^^^
+  |                            `uninitialized` can only be used as the right hand side of a mutable field definition
+-- Error: tests/neg/i11225.scala:10:18 ---------------------------------------------------------------------------------
+10 |  var x6: Int = { uninitialized } // error
+   |                  ^^^^^^^^^^^^^
+   |                  `uninitialized` can only be used as the right hand side of a mutable field definition
```

Comment:
This is on the right hand side of a mutable field definition. The message is inconsistent.

Reply:
The message says `as` not `on`. So I think it's precise.